server/test/unit/vitalsTest/checkButtonHeartbeatTest.js

```diff
@@ -303,9 +315,9 @@ describe('vitals.js unit tests: checkButtonHeartbeat', () => {
 
     describe('when button that is sending vitals last seen exceeds the threshold and no alerts have been sent', () => {
       beforeEach(async () => {
-        this.button = buttonFactory({ sentVitalsAlertAt: null, isSendingVitals: true, client: this.client })
```

Comment:
Completely irrelevant, but every time I see `buttonFactory` I get this song stuck in my head: https://www.youtube.com/watch?v=D8UohwsvBBk